controllers/datadogagent/feature/admissioncontroller/rbac.go

```diff
@@ -0,0 +1,86 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package admissioncontroller
+
+import (
+	rbacv1 "k8s.io/api/rbac/v1"
+
+	"github.com/DataDog/datadog-operator/pkg/extendeddaemonset"
+	"github.com/DataDog/datadog-operator/pkg/kubernetes/rbac"
+)
+
+func getRBACClusterPolicyRules() []rbacv1.PolicyRule {
```

Comment:
Do we have tests for feature RBACs defined somewhere else? I think it's important to cover them, iirc we do test them in the v1alpha1.

Reply:
There aren't individual tests for each feature's specific RBACs but there are general tests for the methods to add ClusterRole, Role, etc.: https://github.com/DataDog/datadog-operator/blob/main/controllers/datadogagent/merger/rbac_test.go. Do you think we should add more feature-specific tests? If so, I can create a card in our backlog since it'll affect a number of features.

Reply:
Sounds good, thanks! Yes, I think there is value in testing we're adding the right permissions required by every feature.